src/test/java/org/opensearch/integTest/SearchMonitorsToolIT.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.integTest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.opensearch.agent.tools.utils.ToolConstants;
+
+import lombok.SneakyThrows;
+
+public class SearchMonitorsToolIT extends BaseAgentToolsIT {
+    private String registerAgentRequestBody;
+    private static final String monitorId = "foo-id";
+    private static final String monitorName = "foo-name";
+
+    @Before
+    @SneakyThrows
+    public void setUp() {
+        super.setUp();
+        registerAgentRequestBody = Files
+            .readString(
+                Path
+                    .of(
+                        this
+                            .getClass()
+                            .getClassLoader()
+                            .getResource("org/opensearch/agent/tools/register_flow_agent_of_search_monitors_tool_request_body.json")
+                            .toURI()
+                    )
+            );
+        createMonitorsSystemIndex(monitorId, monitorName);
+    }
+
+    @After
+    @SneakyThrows
+    public void tearDown() {
+        super.tearDown();
+        deleteExternalIndices();
+        deleteSystemIndices();
+    }
+
+    @SneakyThrows
+    public void testSearchMonitorsToolInFlowAgent_withNoSystemIndex() {
+        deleteSystemIndices();
+        String agentId = createAgent(registerAgentRequestBody);
+        String agentInput = "{\"parameters\":{\"monitorName\": \"" + monitorName + "\"}}";
+        String result = executeAgent(agentId, agentInput);
+        assertEquals("Monitors=[]TotalMonitors=0", result);
+    }
+
+    // TODO: Add IT to test against sample monitor data
+
+    @SneakyThrows
+    private void createMonitorsSystemIndex(String monitorId, String monitorName) {
```

Comment:
We are trying to create the system index for alerting. The best way to do this is to create a dummy monitor, so that the schema mapping of the index is setup correctly and would be better to test for an integration test.

Reply:
Agreed - this was an interim test fn that is actually currently not used. I'll remove and leave the TODO, as this will be grouped with the added tests in the followup issue #136. Thanks for the suggestion, I will likely create dummy monitors and alerts for testing the populated results of the search monitors and search alerts tools respectively.